ui/app/document-upload/controllers/documentController.js

```diff
@@ -15,6 +15,7 @@ angular.module('opd.documentupload')
             var locationUuid = sessionService.getLoginLocationUuid();
 
             $scope.visits = [];
+            $scope.allConcepts = [];
```

Comment:
We can name it better.
Ex: allConcepts to fileTypeConcepts, filteredData to selectedConcept